FIX: Copyedit, typos, style - Remnant: Deep Surveillance (#9628)
endless-sky/endless-sky: data/remnant/remnant 1 introduction.txt

				`	(Don't update the Remnant.)`
					decline
			

> 			`	You approach a spaceport information desk and ask to see Prefect Torza, whom you spoke to the last time you informed the Remnant about the Deep researching the wormhole. Luckily, Torza is on-world at the moment, and you are told to meet him in the cafeteria. When you arrive, you find that Torza has just finished a meal and was about to leave.`

Reviewer: ```suggestion
			`	You approach a spaceport information desk and ask to see Prefect Torza, whom you spoke to the last time you informed the Remnant about the Deep researching the wormhole. Luckily, Torza is on-world at the moment, and you are told to meet him in the cafeteria. When you arrive, you find that Torza has just finished a meal and is about to leave.`
```

			branch "retrieved before"
				"retrieved information" == 2
			

> 			`	You look up the location of Ivan's office that he gave you back when you helped him save the science drone in Terminus, and you find that it is in a building just outside the spaceport - just one room among many in a massive office complex. The complex looks pristine by Dirt Belt standards, but when compared to architecture you've seen elsewhere in the Deep, it is rather brutalistic.`

Reviewer 2: ```suggestion
 
 			`	You look up the location of Ivan's office, which he gave you back when you helped him save the science drone in Terminus, and find that it is in a building just outside the spaceport - just one room among many in a massive office complex. The complex looks pristine by Dirt Belt standards, but when compared to architecture you've seen elsewhere in the Deep, it is rather brutalistic.`
 
 ```

As is, I believe, the comma is not correct to add without also adding the new subject. This adjustment, though, would be an alternative to doing so and still help keep the separate ideas.

> 			`	You look up the location of Ivan's office that he gave you back when you helped him save the science drone in Terminus, and you find that it is in a building just outside the spaceport - just one room among many in a massive office complex. The complex looks pristine by Dirt Belt standards, but when compared to architecture you've seen elsewhere in the Deep, it is rather brutalistic.`

Reviewer 3: Yeah, I think it's complaining about the extra line before and after the text. I do like your version, though.

It's good to see you again.

			`	"Going under the assumption that it is a wormhole that is simply too unstable to allow a ship to pass through, we wanted to see if we could get anything at all to pass through it. What we've discovered is that given a small and smooth enough object, we can get it to disappear within the anomaly without a trace." He then pulls out a small metallic sphere about the size of a ping pong ball from one of the cabinets and dangles it from an attached wire.`
			`	"Now this is only a model, but we created a number of small sensors about this size and sent them through the anomaly. We'd hoped that we could take some readings from whatever is on the other side of the anomaly, as it clearly goes somewhere, but we only ever get static coming back through the wire, followed by nothing a few minutes later. And unfortunately, whatever we send in we can't seem to pull back. The wire always snaps.`

> 			`	"So the anomaly is clearly a wormhole. Now we just need to figure out how to get whatever we send into it to come back and how to get something the size of a ship to enter it."`

Reviewer 3: ```suggestion
			`	"So the anomaly is clearly a wormhole. Now we just need to figure out how to get whatever we send into it to come back, and how to get something the size of a ship to enter it."`
```

Reviewer 2: Losing this comma is correct here, I believe. Just a complex predicate.

			`	"Ivan Haataja? He moved out weeks ago. Said something about 'moving up in this world' while dragging a pallet of filing cabinets out of the building. I can look up where he is now if you need me to."`
			`	"I'd appreciate that," you say.`
			`	The receptionist spends a few minutes tapping away at a keyboard. "Looks like he had his office moved to... the Deep Polytechnic Institute of Valhalla?" he says with a surprised look. "Oh wow, must have gotten someone's attention in the government to be moved there. Do you need directions?"`
			choice
				`	"Yes, please."`
			`	You get directions from the receptionist and thank him for his time. It turns out that this Institute is in the heart of the university district of the spaceport city. Several planets in the Deep, Paradise Worlds, and even many near Earth have a similar university district, but the one on Valhalla is the largest of its kind in the entire Republic.`

> 			`	As you enter the district, you note that the architecture here is much more appealing than the brutalistic style of Ivan's old office building. Each edifice consists of more curves than right angles, with a textured metallic sheen decorating its surface. Brilliant facades scatter sunlight to give the area a warm glow, as opposed to directing it right into your eyes if you happen to be standing in the wrong spot. Even among all the metallic structures, there is still a good amount of greenery: well-trimmed bushes and trees line the streets, and many of the buildings are topped with gardens.`

Author: Note that parts of this paragraph were rewritten. It is not just the addition of `choice` here. Github is not doing a great job highlighting the difference, so please read through it.

> 			`	You ascend the stairs of the Institute, the massive building looming over you as you approach. Its more ancient design makes the building stand out from the rest of the district; the entrance is guarded by massive marble pillars that manage to not clash with the metallic sheen of the rest of the structures nearby. The interior is similarly lavish, with a sprawling lobby area peppered with reception areas and centered by a massive fountain.`

Reviewer: First sentence doesn't feel great - "as you approach" but you just said that you were ascending...

				`	(Approach one of the receptionist desks and ask to see Ivan Haataja.)`
			`	The receptionist acknowledges you with a smile and then taps away at her computer. "It looks like he's available right now," she says. "We'll let him know you're coming and have someone escort you to Doctor Haataja's office shortly."`
			`	A staff member beckons you, and you follow him through what seems like endless hallways and stairwells toward your destination. The entire building seems to be as impressively decorated as the lobby, with beautiful furnishings and fine art. After some time, you finally arrive at Ivan's office. You thank the escort for bringing you here, as you surely would have been lost for hours without him, and knock on Ivan's door.`
			`	Ivan opens the door with a big smile on his face. "Captain <last>! So great to see you here. Please, have a seat." He ushers you inside his office, a large room with a window wall overseeing a scenic courtyard. You sit down in a plush chair on the opposite side of a desk, leaning back to get comfortable. "I know the chairs are cozy," Ivan says with a chuckle. "Don't fall asleep on me, though."`
			branch second
				"retrieved information" == 2

> 			`	He then takes a seat on his own cushioned chair. "Welcome to my office. This is miles above what I'm used to. The discoveries we were making finally attracted the attention of the Deep government, and they immediately had my office moved from that dingy complex it used to be in to here. I have you to thank for much of that work.`

Reviewer: I think "the dingy complex" was better.
```suggestion
			`	He then takes a seat on his own cushioned chair. "Welcome to my office. This is miles above what I'm used to. The discoveries we were making finally attracted the attention of the Deep government, and they immediately had my office moved from the dingy complex it used to be in to here. I have you to thank for much of that work.`
```

				goto reason
			
			label second

> 			`	He then takes a seat on his own cushioned chair. "Welcome to my office. Much better than the old place, isn't it? The discoveries we were making finally attracted the attention of the Deep government, and they immediately had my office moved from that dingy complex it used to be in to here. I have you to thank for much of that work.`

Reviewer: ```suggestion
			`	He then takes a seat on his own cushioned chair. "Welcome to my office. Much better than the old place, isn't it? The discoveries we were making finally attracted the attention of the Deep government, and they immediately had my office moved from the dingy complex it used to be in to here. I have you to thank for much of that work.`
```

					goto suspicion
			branch cubes
				has "Deep: Remnant Surveillance: done"
			`	"Ah. Well, there's quite a bit to talk about." Ivan reclines in his chair.`

> 			`	Ivan pulls out a schematic showing the engines you brought. "The engines you brought us are very interesting. We already mentioned to you how they're rather powerful for their size - more so than even atomic engines of a similar size - while not producing very much heat at all. Their major downside, though, seems to be the immense amount of energy they draw. The energy drain is not surprising given that they seem to be improved ion engines, but the fact that ion engines could ever become more powerful for their size than atomic engines is something spectacular. The thrusters also seem to have the ability to provide reverse thrust, making any ship with them rather maneuverable."`

Reviewer 3: ```suggestion
			`	Ivan pulls out a schematic showing the engines you brought. "The engines you brought us are very interesting. We already mentioned to you how they're rather powerful for their size, more so than even atomic engines of a similar size, while not producing very much heat at all. Their major downside, though, seems to be the immense amount of energy they draw. The energy drain is not surprising given that they seem to be improved ion engines, but the fact that ion engines could ever become more powerful for their size than atomic engines is something spectacular. The thrusters also seem to have the ability to provide reverse thrust, making any ship with them rather maneuverable."`
```

			
			branch skip
				not "Deep: Remnant: Generators: done"

> 			`	He pulls out a schematic showing the generator. "The generator on the other hand wasn't all too interesting. Little more than an improved generator of the type you might find down in the Rim. It is somewhat interesting that it doesn't produce much energy when compared to something like a fusion reactor, given the energy requirement of their engines, but I suppose this people are concerned more with keeping their ships running cool than with having extra power output. Another interesting characteristic we discovered is that the generator seemed to ameliorate the effects of ionization, like that which the Hai Ion Cannons apply, much better than normal. It seemed to be an intentional aspect of the generator's design."`

Reviewer: ```suggestion
			`	He pulls out a schematic showing the generator. "The generator, on the other hand, wasn't all too interesting. Little more than an improved generator of the type you might find down in the Rim. It is somewhat interesting that it doesn't produce much energy when compared to something like a fusion reactor, given the energy requirement of their engines, but I suppose this people are concerned more with keeping their ships running cool than with having extra power output. Another interesting characteristic we discovered is that the generator seemed to ameliorate the effects of ionization, like that which the Hai Ion Cannons apply, much better than normal. It seemed to be an intentional aspect of the generator's design."`
```

			label three
			action
				set "remnant: got old deep research"

> 			`	He hesitates for a moment, looks up at you, and then presses a button on the desk while still looking into your eyes. "Abigail, could you please make a copy of the encrypted folders... 'TAI-RP' on the Institute network under my name and bring them to my office in a data chip? Thank you."`

Reviewer 3: ```suggestion
			`	He hesitates for a moment, looks up at you, then presses a button on the desk while still looking into your eyes. "Abigail, could you please make a copy of the encrypted folders... 'TAI-RP' on the Institute network under my name and bring them to my office in a data chip? Thank you."`
```

> 			`	He hesitates for a moment, looks up at you, and then presses a button on the desk while still looking into your eyes. "Abigail, could you please make a copy of the encrypted folders... 'TAI-RP' on the Institute network under my name and bring them to my office in a data chip? Thank you."`

Reviewer: ```suggestion
			`	He hesitates for a moment, looks up at you, and presses a button on the desk while still looking into your eyes. "Abigail, could you please make a copy of the encrypted folders... 'TAI-RP' on the Institute network under my name and bring them to my office in a data chip? Thank you."`
```
Or do this?

			label four
			action
				set "remnant: got no deep research"

> 			`	He reaches for a button on his desk but then pulls away from it and turns his chair toward the courtyard. "Never mind. I can't give you a copy. That research is classified."`

Reviewer 3: ```suggestion
			`	He is about to press a button on his desk, then pulls away from it and turns his chair toward the courtyard. "Never mind. I can't give you a copy. That research is classified."`
```
I don't think this one is necessary either.

			
			`	Before you can even finish your sentence, you can already feel your muscles tensing up from the taser. You collapse, hitting your head on Ivan's desk as you go down. Before you lose consciousness from the impact, you hear Ivan yell, "Come on! Now I'm going to need to get a new desk."`
			`	You eventually wake up in a cell with handcuffs on. You look around and see a security guard looking at you from outside of the cell. "Ah, you're finally awake." The guard opens the cell. "Doctor Haataja says he isn't going to be pressing charges. Right this way, please."`

> 			`	The guard escorts you out of the cell, which was inside the security office of the Institute, near the lobby. You get some puzzled looks from those in the lobby as you're escorted by the security guard in cuffs. After bringing you outside, the guard unlocks the cuffs. "Don't cause any more trouble, now," he says.`

Reviewer 3: ```suggestion
			`	The guard escorts you out of the cell, which was inside the security office of the Institute near the lobby. You get some puzzled looks from those in the lobby as you're escorted by the security guard in cuffs. After bringing you outside, the guard unlocks the cuffs. "Don't cause any more trouble, now," he says.`
```

				decline
			
			label cubes
			`	Ivan laughs as he reclines in his chair. "Screw that. You should see what I've been able to recover from those sensor cubes you placed.`
				goto ramble
			
			label surveillance
			choice
				`	"I was wondering what came of those sensor cubes I placed."`
				`	"Could I get a copy of what you've found from those sensor cubes?"`
					goto "surveillance copy"
			`	Ivan laughs as he reclines in his chair. "Oh, it's absolutely fascinating what I've been able to find.`
			label ramble

> 			`	"With a cube in each of the Remnant systems, I was able to track all of their fleet movements, and I've discovered some interesting things. For one, ships either make trips between the three systems, or they leave the cluster entirely for significant amounts of time, often at least for a month. There weren't very many ships or fleets that took shorter trips out of the cluster, say only one or two weeks in length. It seems like they're sending people on rather lengthy missions.`

Reviewer 3: ```suggestion
			`	"With a cube in each of the Remnant systems, I was able to track all their fleet movements, and I've discovered some interesting things. For one, ships either make trips between the three systems, or they leave the cluster entirely for significant amounts of time, often at least for a month. There weren't very many ships or fleets that took shorter trips out of the cluster, say only one or two weeks in length. It seems like they're sending people on rather lengthy missions.`
```

> 			`	"With a cube in each of the Remnant systems, I was able to track all of their fleet movements, and I've discovered some interesting things. For one, ships either make trips between the three systems, or they leave the cluster entirely for significant amounts of time, often at least for a month. There weren't very many ships or fleets that took shorter trips out of the cluster, say only one or two weeks in length. It seems like they're sending people on rather lengthy missions.`

Reviewer: ```suggestion
			`	"With a cube in each of the Remnant systems, I was able to track all of their fleet movements, and I've discovered some interesting things. For one, ships either make trips between the three systems, or they leave the cluster entirely for significant amounts of time, often for at least a month. There weren't very many ships or fleets that took shorter trips out of the cluster, say only one or two weeks in length. It seems like they're sending people on rather lengthy missions.`
```
